test/Map/NumField.jl

```diff
@@ -37,6 +40,7 @@
   f = @inferred hom(k, K, a^3)
   @test_throws ErrorException hom(k, K, a)
   @test f(b) == a^3
+  @test_throws AssertionError s = involution(K)
```

Comment:
```suggestion
  @test_throws ArgumentError s = involution(K)
```

Reply:
Is it the same ?

Reply:
No, but `@req` throws an `ArgumentError`.